docs/content/docs/contributing/flutter-code-style-guide.md

```diff
@@ -0,0 +1,78 @@
+## Flutter - code style guide
+
+👉 This style guide describes the preferred style for code written as part of the [`acter` repository](https://github.com/acterglobal/a3)
+
+### Overview
+This document describes our approach to designing and programming Flutter development of the Acter App, from high-level architectural principles all the way to indentation rules.
+
+The primary goal of these style guidelines is to improve code readability so that everyone, whether reading the code for the first time or maintaining it for years, can quickly determine what the code does. Secondary goals are to design systems that are simple, to increase the likelihood of catching bugs quickly, and avoiding arguments when there are disagreements over subjective matters.
+
+That document is focused more about Flutter conventions.
+
+### App Theme
+Use theme classes for the generalise theme of the different components and avoid styling at the component level until and unless it is required.
+
+**Key things to note:**
+- Colors may not be hardcoded in the code, but should be pulled from the AppTheme.
+- Avoid component level styling
+
+Reference: https://docs.flutter.dev/cookbook/design/themes
+
+**Buttons Guide:**
+
+Primary Button
+```
+ ElevatedButton(
+     onPressed: () {},
+    child: ...,
+),
+```
+
+Secondary Button
+```
+ OutlinedButton(
+     onPressed: () {},
+    child: ...,
+),
+```
+
+### Code structure
+There are few basis standard we would like to follow in futter code.
+
+
+**1. General Guide**
+- General try-catches (rather use AsyncValues)
+- Router for state management
+
+**1. Scaffold usage**
+Every parent page should use Scaffold widget to structure entire screen layout.
+
+**2. Scrolling usage**
+- Use `SingleChildScrollView` in all the general use cases where need widgets need to be scrollable
+- Use `CustomScrollView` with its `Slivers` when complex scrolling behavior is required
+
+**3. Avoid code nesting**
+- Use module approach of widget layout and avoid huge nesting in code
+- Create sperate `methods` or `class` for common widgets
```

Comment:
I think examples would be useful. And we said that the rule of thumb is to reduce large nesting and reduce complexity...